packages/atomic/src/components/search/facets/atomic-timeframe-facet/atomic-timeframe-facet.tsx

```diff
@@ -142,6 +142,28 @@ export class AtomicTimeframeFacet
    */
   @MapProp() @Prop() public dependsOn: Record<string, string> = {};
 
+  /**
+   * The earliest date to accept when the `withDatepicker` option is enabled.
```

Comment:
My suggestions are all minor:
```suggestion
   * The earliest date to accept from user input when the `withDatepicker` option is enabled.
```